### ISSUE TITLE
Siblings section skip patterns

### ISSUE THREAD

**Issue by ashreves** [episphere/quest#107]
Was not able to move past the intro for the siblings section. Hitting next did not do anything. Hitting back would take me to the DADCANC2 question (and not the DADCANC3 questions like it should have). Going back and then forward again did not have any effect on getting past siblings page. Refreshing the page took me out of the survey completely. Did put triplet/multiple birth in response to earlier question so did put that I had siblings.

**Comment by danielruss** [comment]
I believe this is related to #86

**Comment by ruggz13** [comment]
If #86 is closed, I will close this @danielruss

**Comment by ashreves** [comment]
@ruggz13 

The skip patterns in the sibling section seem to be an issue.

I had no problems when I entered a simple one sibling, single type of cancer.

When I tried a more complex pattern (multiple siblings, some with cancer and some without), I could not go forward and backwards to change my responses. I got stuck and had to exit the app and restart after deleting my answers. This was tested on a PC using Chrome.
